e-keithel-backend/db/migrate/20220917105702_create_products.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProducts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :products do |t|
+      t.string :name
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
```

Comment:
Nice works introducing the associations. We should introduce more attributes viz.
- name
- description
- image
- price and
- quantity